tests/core/dom/range/getclientrects.js

```diff
@@ -12,6 +12,16 @@
 			this.playground = doc.getById( 'playground' );
 		},
 
+		_should: {
+			ignore: {
+				// Tests randomly fails on FF in Travis
+				'test only element selection': bender.config.isTravis && CKEDITOR.env.gecko,
```

Comment:
You could actually extract `bender.config.isTravis && CKEDITOR.env.gecko` to a variable.